### PR TITLE
feat(docs): Add Discord link to header

### LINKED ISSUE #923: Add link to discord server in docs
Please add our discord server link in header of our docsite

image

### PR DESCRIPTION
**Pull request by pabrahamsson**
resolves #923 

Not terribly familiar with how to generate these docs but this request seemed trivial enough.
I wasn't able to find instructions in the repo for how to build/generate the docs, so any pointers are very welcome.

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -79,6 +79,11 @@ const config = {
             label: 'GitHub',
             position: 'right',
           },
+          {
+            href: 'https://discord.gg/dHJBZw6ewT',
+            label: 'Discord',
```
Comment (christianh814) [inline docs/docusaurus.config.js:84]: Label should be "Discord Community"

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -79,6 +79,11 @@ const config = {
...
+            position: 'right',
```
Comment (christianh814) [inline docs/docusaurus.config.js:85]: Not sure if it matters...I was thinking to position it "left" - but I'm okay with it on the right (not a hill I will die on)

**Review by christianh814 — CHANGES_REQUESTED** [review]
See my comments. I think the only change I'd like is the label.

File: docs/docusaurus.config.js (hunk 2 above)
Comment (krancour) [reply docs/docusaurus.config.js:85]: The GitHub link is on the right, so I think this is exactly where it belongs. Next to it.

**Review by krancour — APPROVED** [review]

**Comment by pabrahamsson** [comment]
> See my comments. I think the only change I'd like is the label.

Thanks @christianh814, I've udpated the PR based on your comment.

**Comment by krancour** [comment]
Thank you @pabrahamsson!

**Pull request merged**